packages/admin/src/pages/super-admin-dashboard/user/[id]/change-roles.page.tsx

```diff
@@ -27,6 +33,9 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       String(role.id)
     );
     const newUserRoles = APPLICANT_ROLES_IDS.concat(body.newUserRoles || []);
+    //case where admin checkbox is disabled (role isn't auto selected)
+    isOwner ? newUserRoles.push('3') : null;
```

Comment:
could we move all the role ids into an object like
```
const ROLE_IDS = {
  FIND: '1',
  APPLICANT: '2',
  ADMIN: '3',
  ...etc
}
```
and use ROLE_IDS.ADMIN instead? same with line 27 & 28 though i know you didnt put that in, magic numbers like this are just a bit rough